OxfordBodleian/Juel-Jensen/BDLaethd22.xml

```diff
@@ -37,9 +37,26 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     <msContents>
                         <summary/>
                         
-                        <!--<msItem xml:id="ms_i1">
-                      
-                        </msItem>-->
+                        <msItem xml:id="ms_i1">
+                            <locus target="#1r"/>
+                            <!--title to be included in index manually-->
+                            <title xml:lang="en">Letter from King <persName ref="PRS9430Tewodros"/> to military officials</title>
+                            <textLang mainLang="am"/>
+                            <note>The following is the entire text. After a greeting to <persName ref="PRS13884Bisawwer"/> and unnamed military leaders, the king informs the recipients
+                                of the arrival of a cannon in <!--or from?--> <placeName ref="LOC2449Dalant"/>. It might be related to the events leading up to the siege of
```

Comment:
I cannot say if the cannon came in or from Dalāntā. Perhaps we may consult Solomon or someone skilled in Amharic.